### PR TITLE
New option in the Online tab: Chat message size

### PR DESCRIPTION
**Pull request by main-gi**
My monitor is quite small, 1338x872 or whatever the hell, and small monitor chat messages take up way too much screen space.

### CODE REVIEW

**Comment by main-gi** [comment]
unknown
unknown
unknown
visuals

And yes, it does update it all when you click between 1-6 in the menu.

File: Marble Blast Platinum/platinum/client/scripts/optionsGui.cs [hunk 1]
```diff
@@ -297,7 +297,7 @@
 $Options::Title   ["Graphics", $i  ] = "Material Quality";
 $Options::Type    ["Graphics", $i  ] = "value";
 $Options::Name    ["Graphics", $i++] = "animateBackground";
-$Options::Title   ["Graphics", $i  ] = "Animate Background";
+$Options::Title   ["Graphics", $i  ] = "Preload Levels";
```
Comment (adierking) [inline Marble Blast Platinum/platinum/client/scripts/optionsGui.cs]: Unrelated change, please remove

File: Marble Blast Platinum/platinum/client/scripts/optionsGui.cs [hunk 2]
```diff
@@ -445,12 +445,15 @@
 $Options::Name    ["Online", 5] = "globalSize";
 $Options::Title   ["Online", 5] = "Global Score Page Size";
 $Options::Type    ["Online", 5] = "value";
-$Options::Name    ["Online", 6] = "autoLogin";
-$Options::Title   ["Online", 6] = "Auto Login";
-$Options::Type    ["Online", 6] = "boolean";
+$Options::Name    ["Online", 6] = "chatMessageSize";
+$Options::Title   ["Online", 6] = "Ingame Lines of Chat";
```
Comment (adierking) [inline Marble Blast Platinum/platinum/client/scripts/optionsGui.cs]: There needs to be a hyphen here, it should be "In-Game Lines of Chat"

**Review by adierking — CHANGES_REQUESTED** [review]
Looks good overall, just a couple minor things.

File: Marble Blast Platinum/platinum/client/scripts/optionsGui.cs (hunk 1 above)
Comment (main-gi) [reply Marble Blast Platinum/platinum/client/scripts/optionsGui.cs]: Whoops. Gonna make a new pull for this, although I don't really know how to deal with multiple changes in the same file with git.

File: Marble Blast Platinum/platinum/client/scripts/optionsGui.cs (hunk 2 above)
Comment (main-gi) [reply Marble Blast Platinum/platinum/client/scripts/optionsGui.cs]: In my opinion "Ingame" looked better, but sure.

**Comment by main-gi** [comment]
Did those.

**Review by adierking — APPROVED** [review]
Awesome!
